From 0085cfb4c4ff6722898e97704a67dcdfcba53388 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Wed, 25 Jan 2017 13:39:08 -0800
Subject: [PATCH 65/67] Ticket 49008 backport 1.3.5 : aborted operation can
 leave RUV in incorrect state

Description: Fixed 2 backport errors in commit 79a3deafe943a3ce5c31c50272939146d17bd7ac.
(cherry picked from commit 3fa6596bdc677cdb3fb65b7baf6fd567485c91a7)
---
 ldap/servers/plugins/replication/csnpl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ldap/servers/plugins/replication/csnpl.c b/ldap/servers/plugins/replication/csnpl.c
index db1ae13..a696fc1 100644
--- a/ldap/servers/plugins/replication/csnpl.c
+++ b/ldap/servers/plugins/replication/csnpl.c
@@ -220,14 +220,14 @@ int csnplCommitAll (CSNPL *csnpl, const CSN *csn)
 	char csn_str[CSN_STRSIZE];
 
 	csn_as_string(csn, PR_FALSE, csn_str);
-	slapi_log_err(SLAPI_LOG_REPL, repl_plugin_name,
+	slapi_log_error(SLAPI_LOG_REPL, repl_plugin_name, 
 		            "csnplCommitALL: committing all csns for csn %s\n", csn_str);
 	slapi_rwlock_wrlock (csnpl->csnLock);
 	data = (csnpldata *)llistGetFirst(csnpl->csnList, &iterator);
 	while (NULL != data)
 	{
 		csn_as_string(data->csn, PR_FALSE, csn_str);
-		slapi_log_err(SLAPI_LOG_REPL, repl_plugin_name,
+		slapi_log_error(SLAPI_LOG_REPL, repl_plugin_name, 
 				"csnplCommitALL: processing data csn %s\n", csn_str);
 		if (csn_is_equal(data->csn, csn) ||
 		    csn_is_equal(data->prim_csn, csn)) {
-- 
2.9.3

